packages/home-spa/src/data.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
Use export as a best practice!

Reply:
removing json and using it as a stub file